src/QueryFilterHelpers.ts

```diff
@@ -60,6 +61,7 @@ import { findClauseInLibrary } from './helpers/ELMHelpers';
  */
 export function parseQueryInfo(
   library: ELM,
+  allELM: ELM[],
```

Comment:
Minor comment here - could you update the `parseQueryInfo()` docstring so that it includes a `@param` tag for `allElm`? This would make it easier to understand what `allElm` represents.

Reply:
Good catch!